web/src/features/panels/LeftPanel.tsx

```diff
@@ -86,6 +86,20 @@ function CollapseButton({ isCollapsed, onCollapse }: CollapseButtonProps) {
   );
 }
 
+function ShareButton() {
+  return (
+    <button
+      data-test-id="share-button"
+      className={
+        'absolute left-full top-3 z-10 flex h-12 w-10 cursor-pointer items-center justify-center rounded-r-xl bg-zinc-50 shadow-[6px_2px_10px_-3px_rgba(0,0,0,0.1)] hover:bg-zinc-100 dark:bg-gray-900 dark:text-gray-400 dark:hover:bg-gray-800'
+      }
+      aria-label={'aria.label.showSidePanel'}
+    >
+      {<Share2Icon />}
```

Comment:
This icon should switch between Apple (iOS and Safari) and non-apple devices, apple should use `<Share>` and non-apple things should use `<Share2>`.

(Drop the Icon part of the import, it's an alias version to the Share2 svg).

The devil is in the details 😈